Sources/RxCBCentral/core/RxPeripheralImpl.swift

```diff
@@ -189,11 +189,14 @@ class RxPeripheralImpl: NSObject, RxPeripheral, CBPeripheralDelegate {
             return
                 didDiscoverServicesSubject
                 .map { (services: [CBService], error: Error?) -> (CBService?, Error?) in
+                    print("RFN ",services.count, services)
+                    print("RFN ",services.first?.uuid.uuidString, service.uuidString)
```

Comment:
Printing is helpful for debugging purposes but we don't want to ship the library with live print statements. Please remove or add these to the logger.